### ISSUE TITLE
Change format to RGBx64

### ISSUE THREAD

**Issue by ThiagoPaiva27** [TheImagingSource/Linux-tiscamera-Programming-Samples#25]
Good morning/afternoon/evening

I was inspecting the TIS.py and noticed that regardless of the sink format chosen, the format BGRa is always used.
I would like to know if there is a way to use the format RGBx64 that was installed with the tiscamerautils, an modification on TIS.py that would allow this change.

Thank you for your attention.

**Comment by TIS-Stefan** [comment]
Hello. 

Sorry for the delay of my answer, I got no notifications from this repository about issues. You are right, RGBx64 is currently not supported. 
I made an enhanced sample.
The change is adding RGB64 to the SinkFormats and added the conversion to numpy. There is a new function, that sets the device-caps into the tcambin, so we can force a video format into the camera. It is used in Program.py after the device is opened. The general sequence is now
```Python
Tis = TIS.TIS()
Tis.open_device("45219983", 1920, 1080, "30/1", TIS.SinkFormats.RGB64, True)
Tis.set_tcamsrc_format("video/x-bayer,format=rggb16")
if not Tis.start_pipeline():  # Start the pipeline so the camera streams
    quit()
```
If RGB64 is used, then the live display will be displayed automatically. 

Sample code: [device-caps.zip](https://github.com/TheImagingSource/Linux-tiscamera-Programming-Samples/files/10802254/device-caps.zip)

Stefan
